en/modules/modules.xml

```diff
@@ -523,31 +506,42 @@
             
         </section>
         
-        <section id="docarchives">
-            <title>Documentation archives</title>
+        <section id="docfolder">
+            <title>Documentation folder</title>
             
-            <para>A <emphasis>documentation archive</emphasis> is a <literal>zip</literal> 
-            archive which:</para>
+            <para>A <emphasis>documentation folder</emphasis> is a folder which:</para>
             
             <itemizedlist>
                 <listitem>
                     <para>contains the module documentation generated by the documentation
-                    compiler (<literal>.html</literal> and resources files), and</para>
+                    compiler (<literal>.html</literal> and resources files) in the
+                    <literal>api</literal> folder,</para>
                 </listitem>
                 <listitem>
-                    <para>has a filename which adheres to the standard for
-                    documentation archive names.</para>
+                    <para>can contain any user documentation aggregated by the documentation
+                    compiler (any files that were found in the project's <literal>doc</literal>
+                    folder) in the <literal>doc</literal> folder, and</para>
+                </listitem>
+                <listitem>
+                    <para>has the name <literal>module-doc</literal>.</para>
                 </listitem>
             </itemizedlist>
             
-            <para>Inside a documentation archive, HTML source is located in the 
-            <literal>module-doc</literal> directory.</para>
+            <para>Inside a documentation folder, HTML source for the API documentation
+            is located in the <literal>api</literal> directory.</para>
+            
+            <para>Inside a documentation folder, any files related to user documentation
+            are located in the <literal>doc</literal> directory.</para>
             
             <para>Thus, the structure of the documentation archive for the module
```

Comment:
Yes, documents, like resources, are stored as a bunch of files in a folder. Only for transfer purposes to/from a remote server like the Herd are they zipped.